main.py

```
@@ -441,7 +449,8 @@ async def cli_loop(sk, pk, chain, mempool, network):
                 continue
             peer_id = parts[1]
             from minichain.persistence import ban_peer
-            ban_peer(peer_id, reason="Manual ban via CLI")
+            ban_peer(peer_id, reason="Manual ban via CLI", path=datadir or ".")
+            asyncio.create_task(network.disconnect_peer(f"peer:{peer_id}"))
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)